src/entity/Ship.java

```diff
@@ -50,8 +50,8 @@ public class Ship extends Entity {
 	 *            Initial position of the ship in the Y axis.
 	 */
 	//Edit by Enemy
-	public Ship(final int positionX, final int positionY) {
-		super(positionX, positionY - 50, 13 * 2, 8 * 2, Color.GREEN);
+	public Ship(final int positionX, final int positionY, final Color color) {
```

Comment:
I think it’s a good idea to add a color variable to the Ship class constructor, allowing you to customize the color of the ship as desired.

Reply:
Thank you.